test/cctest/test-assembler-riscv64.cc

```diff
@@ -572,7 +572,7 @@ UTEST_LOAD_STORE(lw, sw, int32_t, 0x456AF894)
 // set the 32th least significant bit of
 // value-to-store to 1 to test
 // zero-extension by lwu
-UTEST_LOAD_STORE(lwu, sw, int32_t, 0x856AF894)
+UTEST_LOAD_STORE(lwu, sw, int32_t, 0x756AF894)
```

Comment:
The point of this test case is to verify that it zero-extends, so changing the input to not have a ` at bit 31 voids the test case. Instead, does the 3rd parameter, `int32_t` need to be changed to `uint32_t`. @penguinwu, can you take a look at this one?

Reply:
The intension is to test the store and load of `0x856AF894` as `uint32_t`. The most significant bit of the `uint32_t` is deliberately set to make sure `lwu` does zero-extension. So the fix should be changing the test value.

I wonder if using `int32_t` in the macro instead of `uint32_t` could be the culprit. I think we used `uint32_t` because the template for simplicity does not support unsigned integer types.

Let me do a refactoring to support unsigned integer types in UTEST_LOAD_STORE first to see if it helps the problem.

Reply:
yes, refact the UTEST_LOAD_STORE  is best way. So i close the pr.

Reply:
I found wo can only add one function for `SetParam(Param_T* params, uint32_t val)`. Please see  #44.